docker-compose.yml

```diff
@@ -158,53 +153,176 @@ services:
+      - CONTACT_US_URL=https://frontend.levellingup.gov.localhost:3008/contact_us
+      - FEEDBACK_LINK=https://frontend.levellingup.gov.localhost:3008/feedback
+      - COOKIE_POLICY_URL=https://frontend.levellingup.gov.localhost:3008/cookie_policy
+      - ACCESSIBILITY_STATEMENT_URL=https://frontend.levellingup.gov.localhost:3008/accessibility_statement
+      - SERVICE_START_PAGE=https://frontend.levellingup.gov.localhost:3008/account
+      - MULTIFUND_URL=https://frontend.levellingup.gov.localhost:3008/account
+      - LOGOUT_URL=https://authenticator.levellingup.gov.localhost:3004/sessions/sign-out
+      - PRIVACY_POLICY_URL=https://frontend.levellingup.gov.localhost:3008/privacy
+      - ELIGIBILITY_RESULT_URL=https://frontend.levellingup.gov.localhost:3008/eligibility-result
     env_file: .awslocal.env
+    networks:
+      default:
+        aliases:
+          - form-runner.levellingup.gov.localhost
+    profiles: [ pre ]
+
+  authenticator:
+    build:
+      context: ./apps/funding-service-design-authenticator
+    volumes:
+      - './apps/funding-service-design-authenticator:/app'
+      - './certs:/app-certs'
+    command: bash -c "python -m debugpy --listen 0.0.0.0:5678 -m flask run -p 8080 -h 0.0.0.0 --cert=/app-certs/cert.pem --key=/app-certs/key.pem"
+    ports:
+      - 3004:8080
+      - 5684:5678
+    depends_on:
+      - redis-data
+    env_file:
+      - .env # For the AZURE_AD_* variables
+      - .awslocal.env
+    environment:
+      - REDIS_INSTANCE_URI=redis://redis-data:6379
+      - ACCOUNT_STORE_API_HOST=http://account-store:8080
+      - APPLICATION_STORE_API_HOST=http://application-store:8080
+      - FUND_STORE_API_HOST=http://fund-store:8080
+      - AUTHENTICATOR_HOST=https://authenticator.levellingup.gov.localhost:3004
+      - NOTIFICATION_SERVICE_HOST=http://notification:8080
+      - APPLICANT_FRONTEND_HOST=https://frontend.levellingup.gov.localhost:3008
+      - ASSESSMENT_FRONTEND_HOST=https://assessment.levellingup.gov.localhost:3010
+      - POST_AWARD_FRONTEND_HOST=https://find-monitoring-data.levellingup.gov.localhost:4001
+      - POST_AWARD_SUBMIT_HOST=https://submit-monitoring-data.levellingup.gov.localhost:4001
+      - COOKIE_DOMAIN=.levellingup.gov.localhost
+    networks:
+      default:
+        aliases:
+          - authenticator.levellingup.gov.localhost
+
+  notification:
+    build:
+      context: ./apps/funding-service-design-notification
+    volumes: [ './apps/funding-service-design-notification:/app' ]
+    command: [ "sh", "-c", "python -m debugpy --listen 0.0.0.0:5678 -m wsgi" ]
+    ports:
+      - 3006:8080
+      - 5686:5678
+    env_file: .awslocal.env
+    environment:
+      - GOV_NOTIFY_API_KEY=${GOV_NOTIFY_API_KEY:?err}
+      - FLASK_DEBUG=1
+      - FLASK_ENV=development
+      - AWS_MSG_BUCKET_NAME=fsd-notification-bucket
+    profiles: [ pre ]
+
+  data-store:
+    # http://github.com/communitiesuk/funding-service-design-post-award-data-store
```

Comment:
V minor point but this feels like a handy thing to link to the GitHub repo where the service is declared in the docker-compose, could be nice to add it for the rest of them?

Reply:
yep yep, will sneak this in as a followup PR, good shout